contracts/core/BasePositionManager.sol

```diff
@@ -162,36 +171,30 @@ contract BasePositionManager is IBasePositionManager, ReentrancyGuard, Governabl
         if (_isLong) {
             uint256 maxGlobalLongSize = maxGlobalLongSizes[_indexToken];
             if (maxGlobalLongSize > 0 && IVault(vault).guaranteedUsd(_indexToken).add(_sizeDelta) > maxGlobalLongSize) {
-                revert("BasePositionManager: max global longs exceeded");
+                revert("max longs exceeded");
```

Comment:
it is to decrease size of the contract, otherwise it won't compile?

Reply:
yes